src/simulator/api.yaml

```diff
@@ -2124,15 +2124,20 @@ components:
       properties:
         isValid:
           type: boolean
-        authChannels:
-          type: array
-          items:
-            $ref: '#/components/schemas/ConsentRequestChannelType'
-        authUri:
-          type: string
-          description: >-
-            The callback uri that the pisp app redirects to for user to complete
-            their login.
+        data:
+          type: object
+          properties:

```

Comment:
are `authChannles` and `authUri` not required?

Reply:
generally we are not specifying any mandatory fields in simulator API specification, response is based on rules